graf2d/asimage/src/TASImage.cxx

```diff
@@ -1095,7 +1095,8 @@ void TASImage::FromPad(TVirtualPad *pad, Int_t x, Int_t y, UInt_t w, UInt_t h)
 
    if (gROOT->IsBatch()) { // in batch mode
       TVirtualPS *psave = gVirtualPS;
-      gVirtualPS = (TVirtualPS*)gROOT->ProcessLineFast("new TImageDump()");
+      const static auto tclass_TImageDump = TClass::GetClass("TImageDump");
+      gVirtualPS = (TVirtualPS *) tclass_TImageDump->GetNew()(nullptr);
```

Comment:
The original implementation had `TClass::New` however, I backed off since that method does a bit more than what the `new` does in the dictionaries: see [here](https://root.cern.ch/doc/master/TClass_8cxx_source.html#l05091). One of the operations worth noting is for example invoking [RegisterAddressInRepository](https://root.cern.ch/doc/master/classTClass.html#abd11cc76850cdd0c024436fa88497943).

Clearly using `TClass::New` is simpler from the point of view of this particular change, but are we sure it does not have undesired side effects?